_knitr/A Shiny app for Hong Kong District Councillors_20200801.md

```diff
@@ -9,42 +9,70 @@ output:                    # DO NOT CHANGE
+You can access:
 
-## Background
+- The app [here](https://hkdistricts-info.shinyapps.io/dashboard-hkdistrictcouncillors/). 
+- Our GitHub repository [here](https://github.com/avisionh/dashboard-hkdistrictcouncillors). 
+- Don't forget to also provide some feedback to the Shiny app [here](https://hkdistrictsinfo.typeform.com/to/gFHC02gE)!
 
-- There are heightened levels of political engagement in Hong Kong in the recent years, which is leading to a surge in demand for information and knowledge about how people can make a difference. 
-- The District Council is the most local level of government in which people can influence change.
-- Who we are
+Whether you are more of an R enthusiast or simply someone who has an interest in Hong Kong politics (hopefully both!), we hope this post will bring you some inspiration on how you can use R _not just_ as a great tool for data analysis, but also as an enabler for you to do something tangible for your community and contribute to causes you care about. 
+
+## Why develop this app?
+There are heightened levels of political engagement in Hong Kong in the recent years, which is leading to a surge in demand for information and knowledge about how people can make a difference. 
+
+The District Council is the most local level of government in which people can influence change.
+
+## Who is behind this?
 - Avision Ho, a data scientist whom I had [previously interviewed.](https://martinctc.github.io/blog/data-chats-an-interview-with-avision-ho/)
-- This project is an attempt to help make a difference with R programming. It's an opportunity for us to learn, to code, to have fun, and to make a difference.
+- Ocean
+- Justin
+- Tiffany
+
+## What is in the app?
 
-## The App
+The Shiny app is built like a dashboard which combines information about each district councillor alongside their Facebook page posts (if it exists) and the district they serve, illustrated on an interactive map.
 
-<Describe the layout of the app>
+Specifically, there are several key components that were used on top of the incredible [shiny](https://github.com/rstudio/shiny) package:
 
-## Data Collection and Cleaning
+- [shinydashboard](https://github.com/rstudio/shinydashboard): For mobile-friendly dashboard layout.
+- [googlesheets4](https://github.com/tidyverse/googlesheets4): For seamless access to GoogleSheets.
+- [sf](https://github.com/r-spatial/sf) and [leaflet](https://github.com/rstudio/leaflet): For importing geographic data and creating interactive maps.
+- [rintrojs](https://github.com/carlganz/rintrojs): For interactive tutorials
+
+## How was the data collected?
 <Insert screenshot>
-- Where the data came from
-- Deliberated to using Google Sheets to avoid bulking up the repo, and also enable participation from those who do not code
+- Data on each district councillor, their constituency, the party they belong to and their Facebook page was all collected manually through a combination of Wikipedia and Facebook.
```

Comment:
@ocean0609 to provide further details of this process.

Reply:
I was thinking of a separate blog post for the package and a light reference to that here. What do you think?